neuraxle/steps/loop.py

```diff
@@ -241,3 +255,120 @@ def transform(self, data_inputs: List) -> List:
 
     def inverse_transform(self, data_output):
         return [self.steps[i].inverse_transform(di) for i, di in enumerate(data_output)]
+
+
+class FlattenForEach(ResumableStepMixin, MetaStepMixin, BaseStep):
+    """
+    Step that reduces a dimension instead of manually looping on it.
```

Comment:
how does it reduces? which axises are merged together? please documetn. DOes it merge axis 0 and 1, or others?